test_runner/fixtures/zenith_fixtures.py

```diff
@@ -431,6 +431,10 @@ def __init__(self, config: ZenithEnvBuilder):
 pg_port = {pageserver_port.pg}
 http_port = {pageserver_port.http}
 auth_type = '{pageserver_auth_type}'
+gc_period = '1s'
+gc_horizon = 67108864
+checkpoint_period = '1s'
+checkpoint_distance = 1048576
```

Comment:
Using non-default settings in all tests doesn't look right to me.
Let's tune them for one test, if necessary. Or change defaults if we believe that they are bad.

Reply:
I'd argue the change doesn't belong to this PR.

Reply:
This changes are needed for test_recovery.py test included in this PR.